monitored-httpclient/src/main/java/com/github/nhenneaux/resilienthttpclient/monitoredclientpool/SingleIpHttpClient.java

```diff
@@ -117,10 +117,10 @@ void checkHealthStatus() {
                 httpRequestBuilder.timeout(Duration.ofMillis(serverConfiguration.getHealthReadTimeoutInMilliseconds()));
             }
 
-            Consumer<HttpRequest.Builder> requestTransformer = serverConfiguration.getRequestTransformer() != null ? serverConfiguration.getRequestTransformer() : builder -> {
-            };
-
-            requestTransformer.accept(httpRequestBuilder);
+            final Consumer<HttpRequest.Builder> requestTransformer = serverConfiguration.getRequestTransformer();
```

Comment:
You can set ServerConfiguration.DEFAULT_REQUEST_TRANSFORMER to null as well 
https://github.com/nhenneaux/resilient-httpclient/blob/master/single-host-httpclient/src/main/java/com/github/nhenneaux/resilienthttpclient/singlehostclient/ServerConfiguration.java#L15

Reply:
So, that means in ah-core I will keep the HttpClientConfig for getHealthCheckRequestTransformer to return null?

Reply:
yes